wavelength-plugins/friends.js

```diff
@@ -7,10 +7,19 @@
 
 function onlineFriends(user) {
 	let list = Db.friends.get(user, []).filter(friend => Users(friend) && Users(friend).connected);
-	if (list.length) Users(user).send(`|pm| WL Friend Manager|~|You have ${list.length} friends online: ${Chat.toListString(list)}`);
+	if (list.length && !Db('stopFriendNotifs').get(user)) Users(user).send(`|pm| WL Friend Manager|~|You have ${list.length} friends online: ${Chat.toListString(list)}`);
```

Comment:
Users(user) is redudant I'm pretty sure.
That Db is incorrect it would be `Db.stopFriendNotifs.get(user)` but wouldn't `Db.stopFriendNotifs.has(user)` be more proper?

Reply:
every time i've used .has it glitches so i'm not going to use it

Reply:
``Users(user) is redudant I'm pretty sure.``
W
R
O
N
G

that is set as the id of the user not the object so, wrong

Reply:
but yeah that is the wrong Db kek